k8s/perftest/common/probate/tests-values.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: tests-values
```

Comment:
you can't just copy these values in they need to be re-encrypted. Although not sure if you need it so I would remove it